PyRDF/__init__.py

```diff
@@ -5,19 +5,21 @@
 from .CallableGenerator import CallableGenerator
 from .backend import Local, Dist
 
-Proxy.backend = Local()
+current_backend = Local()
 
 def use(backend, conf = {}):
 
     future_backends = [
     "dask"
     ]
 
+    global current_backend
+
     if backend in future_backends:
         raise NotImplementedError(" This backend environment will be considered in the future !")
     elif backend == "local":
```

Comment:
and I would change the name of this variable to `backend_name`

Reply:
@etejedor , `current_backend` is the same global variable that I'd declared outside. I have to use the `global` keyword to access it. Otherwise, the use function consider it as a variable in its scope. So, shall I rename this to `backend` itself as you'd suggested then ?

Reply:
And also @etejedor , there would be an issue in importing `backend` if I change the name of the global variable to backend. There would be a conflict between this global variable and the sub-package named backend

Reply:
Then I would change `backend` to `backend_name` and keep `current_backend` as it is.

Reply:
👍